docs/source/tutorials/e2e_flow.rst

```diff
@@ -25,222 +25,239 @@ from the community seemlessly with torchtune.
 
 .. _eval_harness_label:
 
-Run Evaluation using EleutherAI's Eval Harness
-----------------------------------------------
-
-We've fine-tuned a model. But how well does this model really do? Let's run some Evaluations!
+Run evals using EleutherAI's Eval Harness
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. TODO (SalmanMohammadi) ref eval recipe docs
 
 torchtune integrates with
 `EleutherAI's evaluation harness <https://github.com/EleutherAI/lm-evaluation-harness>`_.
 An example of this is available through the
-``eleuther_eval`` recipe. In this tutorial, we're going to directly use this recipe by
-modifying its associated config ``eleuther_evaluation.yaml``.
+`eleuther_eval <https://github.com/pytorch/torchtune/blob/main/recipes/eleuther_eval.py>`_ recipe. In this tutorial, we're going to directly use this recipe by
+modifying its associated config `eleuther_evaluation.yaml <https://github.com/pytorch/torchtune/blob/main/recipes/configs/eleuther_evaluation.yaml>`_.
 
 .. note::
-    For this section of the tutorial, you should first run :code:`pip install lm_eval==0.4.*`
+    For this section of the tutorial, you should first run :code:`pip install lm_eval>=0.4.5`
     to install the EleutherAI evaluation harness.
 
 Since we plan to update all of the checkpoint files to point to our fine-tuned checkpoints,
-let's first copy over the config to our local working directory so we can make changes. This
-will be easier than overriding all of these elements through the CLI.
+let's first copy over the config to our local working directory so we can make changes.
 
 .. code-block:: bash
 
-    tune cp eleuther_evaluation ./custom_eval_config.yaml \
+    $ tune cp eleuther_evaluation ./custom_eval_config.yaml
+    Copied file to custom_eval_config.yaml
 
-For this tutorial we'll use the `truthfulqa_mc2 <https://github.com/sylinrl/TruthfulQA>`_ task from the harness.
-This task measures a model's propensity to be truthful when answering questions and
-measures the model's zero-shot accuracy on a question followed by one or more true
-responses and one or more false responses. Let's first run a baseline without fine-tuning.
-
-
-.. code-block:: bash
-
-    tune run eleuther_eval --config ./custom_eval_config.yaml
-    checkpointer.checkpoint_dir=<checkpoint_dir> \
-    tokenizer.path=<checkpoint_dir>/tokenizer.model
-
-    [evaluator.py:324] Running loglikelihood requests
-    [eleuther_eval.py:195] Eval completed in 121.27 seconds.
-    [eleuther_eval.py:197] truthfulqa_mc2: {'acc,none': 0.388...
+Then, in your config, you only need to replace two fields: ``output_dir`` and ``checkpoint_files``. Notice
+that we are using the merged weights, and not the LoRA adapters.
 
-The model has an accuracy around 38.8%. Let's compare this with the fine-tuned model.
+.. code-block:: yaml
 
+    # TODO: update to your desired epoch
+    output_dir: /tmp/torchtune/llama3_2_3B/lora_single_device/epoch_0
```

Comment:
if we do have epoch_0 and epoch_1, then maybe we should update all of the paths to epoch_1. I originally did epoch_0 to be on the safe side